### PR TITLE
Dịch "weight-decay_vn" - Phần 3

### PR DESCRIPTION
**Pull request by nguyenduydu**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [x] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_multilayer-perceptrons/weight-decay_vn.md [hunk 1]
```diff
@@ -161,28 +171,32 @@ Often, we do not regularize the bias term of a network's output layer.
...
 
 <!--
 choosing our label to be a linear function of our inputs, corrupted by Gaussian noise with zero mean and variance 0.01.
 To make the effects of overfitting pronounced, we can increase the dimensinoality of our problem to $d = 200$
 and work with a small training set containing only 20 example.
 -->
 
-*dịch đoạn phía trên*
+lựa chọn nhãn là một hàm tuyến tính của các đầu vào, bị biến dạng bởi nhiễu Gauss với trung bình bằng không và độ lệch chuẩn bằng 0.01.
+Để làm cho hiệu ứng quá khớp rõ ràng, ta có thể tăng số chiều của bài toán lên $d = 200$ và làm việc với một tập huấn luyện nhỏ bao gồm chỉ 20 mẫu.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/weight-decay_vn.md]:
```suggestion
Để làm cho hiệu ứng của việc quá khớp trở nên rõ ràng, ta có thể tăng số chiều của bài toán lên $d = 200$ và làm việc với một tập huấn luyện nhỏ bao gồm chỉ 20 mẫu.
```

File: chapter_multilayer-perceptrons/weight-decay_vn.md [hunk 2]
```diff
@@ -151,7 +154,14 @@ Whether we include a corresponding bias penalty $b^2$ can vary across implementa
 Often, we do not regularize the bias term of a network's output layer.
 -->
 
-*dịch đoạn phía trên*
+Như trước đây, ta cập nhật $\mathbf{w}$ dựa trên một lượng bằng sự khác biệt của ước lượng và mẫu.
+Tuy nhiên, ta sẽ nén độ lớn của $\mathbf{w}$ về phía $0$.
+Đó là lý do tại sao phương pháp này còn đôi khi được gọi là "phân rã trọng số": giữ nguyên hệ số phạt, thuật toán tối ưu của ta *phân rã* trọng số ở từng bước của quá trình huấn luyện.
+Ngược lại so với lựa chọn đặc trưng, phân rã trọng số cho ta một cơ chế liên tục cho việc thay đổi độ phức tạp của $f$.
+Các giá trị nhỏ của $\lambda$ tương ứng với $\mathbf{w}$ không bị nén, trong khi các giá trị lớn của $\lambda$ nén $\mathbf{w}$ một cách đáng kể.
```
[the remark this answers is not in the document]
Comment (nguyenduydu) [reply chapter_multilayer-perceptrons/weight-decay_vn.md]: Ban đầu mình cũng dịch là 'ràng buộc' nhưng sau đó nghĩ nên để là 'nén' để consistent với đoạn bên trên. Mình đồng ý là 'ràng buộc' hay hơn :). Cám ơn bạn.

File: chapter_multilayer-perceptrons/weight-decay_vn.md [hunk 3]
```diff
@@ -161,28 +171,32 @@ Often, we do not regularize the bias term of a network's output layer.
...
-## *dịch tiêu đề phía trên*
+## Hồi quy tuyến tính nhiều chiều
 
 <!--
 We can illustrate the benefits of weight decay over feature selection through a simple synthetic example.
 First, we generate some data as before
 -->
 
-*dịch đoạn phía trên*
+Ta có thể minh họa các lợi ích của phân rã trọng số so với lựa chọn đặc trưng thông qua một ví dụ tổng hợp đơn giản.
```
[the remark this answers is not in the document]
Comment (nguyenduydu) [reply chapter_multilayer-perceptrons/weight-decay_vn.md]: Mình cũng thấy 'dữ liệu giả' hợp lý hơn nhưng vì mình thấy ở các phần dịch trước đó mọi người cũng dùng từ 'tổng hợp' nên mình dùng theo.

File: chapter_multilayer-perceptrons/weight-decay_vn.md [hunk 4]
```diff
@@ -151,7 +154,14 @@ Whether we include a corresponding bias penalty $b^2$ can vary across implementa
 Often, we do not regularize the bias term of a network's output layer.
 -->
 
-*dịch đoạn phía trên*
+Như trước đây, ta cập nhật $\mathbf{w}$ dựa trên một lượng bằng sự khác biệt của ước lượng và mẫu.
+Tuy nhiên, ta sẽ nén độ lớn của $\mathbf{w}$ về phía $0$.
+Đó là lý do tại sao phương pháp này còn đôi khi được gọi là "phân rã trọng số": giữ nguyên hệ số phạt, thuật toán tối ưu của ta *phân rã* trọng số ở từng bước của quá trình huấn luyện.
+Ngược lại so với lựa chọn đặc trưng, phân rã trọng số cho ta một cơ chế liên tục cho việc thay đổi độ phức tạp của $f$.
```
[the remark this answers is not in the document]
Comment (nguyenduydu) [reply chapter_multilayer-perceptrons/weight-decay_vn.md]: Mình sẽ thay thế `Ngược lại với` trong đoạn gợi ý của bạn bằng cụm từ `Trái ngược với`. Cám ơn bạn nhé.

**Comment by nguyenduydu** [comment]
@minhduc0711  Cám ơn bạn đã góp ý nhé.

File: chapter_multilayer-perceptrons/weight-decay_vn.md [hunk 5]
```diff
@@ -127,13 +127,16 @@ In practice, this might make them more robust to measurement error in a single v
 By contrast, L1 penalties lead to models that concentrate weight on a small set of features, which may be desirable for other reasons.
 -->
 
-*dịch đoạn phía trên*
+Một lý do để sử dụng chuẩn L2 là vì nó phạt rất nặng những thành phần lớn của véc-tơ trọng số.
```
Comment (duythanhvn) [inline chapter_multilayer-perceptrons/weight-decay_vn.md]: Uầy, bình tĩnh nào...
```suggestion
Một lý do để sử dụng chuẩn L2 là vì nó phạt rất nặng những thành phần lớn của vector trọng số.
```

File: chapter_multilayer-perceptrons/weight-decay_vn.md [hunk 6]
```diff
@@ -161,28 +171,32 @@ Often, we do not regularize the bias term of a network's output layer.
 ## High-Dimensional Linear Regression
 -->
 
-## *dịch tiêu đề phía trên*
+## Hồi quy tuyến tính nhiều chiều
```
Comment (duythanhvn) [inline chapter_multilayer-perceptrons/weight-decay_vn.md]:
```suggestion
## Hồi quy Tuyến tính nhiều chiều
```

File: chapter_multilayer-perceptrons/weight-decay_vn.md (hunk 5 above)
Comment (nguyenduydu) [reply chapter_multilayer-perceptrons/weight-decay_vn.md]: @duythanhvn  :) để mình sưa lại các bản dịch  khác nữa. Mình nhầm tưởng anh em thống nhất dịch `vector `là  `véc-tơ`. Vừa kiểm tra lại bảng glossary.  Thanks.

File: chapter_multilayer-perceptrons/weight-decay_vn.md [hunk 7]
```diff
@@ -151,7 +154,14 @@ Whether we include a corresponding bias penalty $b^2$ can vary across implementa
 Often, we do not regularize the bias term of a network's output layer.
 -->
 
-*dịch đoạn phía trên*
+Như trước đây, ta cập nhật $\mathbf{w}$ dựa trên một lượng bằng sự khác biệt của ước lượng và mẫu.
```
[the remark this answers is not in the document]
Comment (duythanhvn) [reply chapter_multilayer-perceptrons/weight-decay_vn.md]: Bạn trẻ miss 2 suggestions hay sao nhỉ?

[the remark this answers is not in the document]
Comment (minhduc0711) [reply chapter_multilayer-perceptrons/weight-decay_vn.md]: chắc bạn ý quên không expand mấy cái bị hidden
@nguyenduydu bạn xem hộ mình nốt mấy suggestions này được không, thanks

**Comment by nguyenduydu** [comment]
@minhduc0711 mình k để ý mấy cái expand. Thanks bạn nhé.
